### PR TITLE
Fix grammatical mistake in role assignment message

### PR DESCRIPTION
**Pull request by Pranjal-Patel**
**What kind of change does this PR introduce?**  
grammatical mistake fix

### CODE REVIEW

File: bot/events/on_message.py [hunk 1]
```diff
@@ -39,7 +39,7 @@ async def addToRole(message, role_name, role_color):
                             break
                     # Congradulates user :D
                     await message.channel.send(
-                        f"Congratulations {message.author.mention}, you are now an {role_name}!"
+                        f"Congratulations {message.author.mention}, you are now {'a' if role_name == 'Member' else 'an'} {role_name}!"
```
Comment (timelessnesses) [inline bot/events/on_message.py:42]: might be better if
```py
'a' if role_name[0] not in "aeiou" else "an"
```

Comment (Pranjal-Patel) [reply bot/events/on_message.py]: done

**Review by timelessnesses — APPROVED** [review]
lgtm

**Comment by logicguy1** [comment]
Looks good, mergereing

**Pull request merged**